src/tasks/Promotions/YouTube/Routes.ts

```diff
@@ -0,0 +1,64 @@
+import * as express from "express";
+import { channelIds } from "../../../channel-IDs";
+
+export const youtube = express.Router();
+
+youtube.get('/notification', ({ query: { 'hub.challenge': challenge } }, res) => {
+  res.status(200).end(challenge);
+});
+
+youtube.post('/notification', ({ body, app }, res) => {
+  try {
+    const publications = body.feed.entry;
+    
+    if (!publications) {
+      throw new Error('Request without publication');
+    };
+    
+    const discordClient = app.get('discordClient');
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const parsePublications = (pub: any) => {
```

Comment:
if you want to keep the linter from complaining, you can use `unknown` instead of `any`.

Reply:
<img width="414" alt="Screenshot 2021-08-31 at 21 57 06" src="https://user-images.githubusercontent.com/41327204/131567713-e710e465-bd8b-4cf7-9dbc-ca455c097b5b.png">

Reply:
ahh, try using with optional chaining

pub?.author?.[0] etc etc

Reply:
with type unknown, chaining doesn't help. I made interface { [key:string]: unknown } and it works fine